src/state/migrations/index.ts

```diff
@@ -84,6 +84,8 @@ export const clearAssetsMigrations = {
   52: clearAssets,
   53: clearAssets,
   54: clearAssets,
+  55: clearAssets,
+  56: clearAssets,
```

Comment:
It doesn't really matter, but we should only need this once.

Reply:
You're too fast flash, it's pending one more asset gen, I'll revert when the draft mode will be turned off

Reply:
Reverted in [9049b18](https://github.com/shapeshift/web/pull/9144/commits/9049b1851440e3ee1ab8315daf30b40d1b748065)